sdk/tests/deqp/framework/referencerenderer/rrRenderer.js

```diff
@@ -386,7 +386,7 @@ void FragmentProcessor::render (const rr::MultisamplePixelBufferAccess& msColorB
     var stencilState = state.stencilStates[fragmentFacing];
     var colorMaskFactor = [state.colorMask[0] ? 1 : 0, state.colorMask[1] ? 1 : 0, state.colorMask[2] ? 1 : 0, state.colorMask[3] ? 1 : 0];
     var colorMaskNegationFactor = [state.colorMask[0] ? false : true, state.colorMask[1] ? false : true, state.colorMask[2] ? false : true, state.colorMask[3] ? false : true];
-    var sRGBTarget = false;
+    var sRGBTarget = state.sRGBEnabled && colorBuffer.getFormat().isSRGB();
 
```

Comment:
See the native c++ code: https://android.googlesource.com/platform/external/deqp/+/deqp-dev/framework/referencerenderer/rrFragmentOperations.cpp#777. 

Some files between JS and native C++ are not exactly the same, but this function in JS (rrRenderer.js) is ported from the one in rrFragmentOperation.cc.